src/main/java/kr/allcll/seatfinder/pin/PinService.java

```diff
@@ -0,0 +1,29 @@
+package kr.allcll.seatfinder.pin;
+
+import java.util.List;
+import kr.allcll.seatfinder.subject.Subject;
+import kr.allcll.seatfinder.subject.SubjectRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class PinService {
+
+    private final PinRepository pinRepository;
+    private final SubjectRepository subjectRepository;
+
+    public void addPinAtSubject(Long subjectId, String userToken) {
```

Comment:
사용자라는 정보를 포함하지 않아도 될 것 같아요! 

```suggestion
public void addPinAtSubject(Long subjectId, String token) {
```

Reply:
s로 복수를 챙긴다거나, 메서드명을 상세히 적는다던가 하는 건 좋아하셨던 것 같은데용,,, 혹시 어떤걸 명확히 상세하게 명시하는 걸 좋아하고, 어떤걸 일반적으로 명명하는게 좋다고 생각하는지 기준이 궁금해요!!
알려주시면 이후에도 이런 식으로 수정하는 리소스가 줄 것 같고, 주환님도 네이밍 관련해서 코멘트 계속 남기는 리소스가 줄 것 같아용

Reply:
언급해주신 부분에 대한 합의 없이, 제가 일방적으로 주장한 것 같아요.. 미안합니다. 언급해주신 부분에 대해서 어떠한 명확한 기준을 정하기 어려운 것 같아요. 제가 코멘트를 달 때 의도를 확실히 작성해 볼게요! 그 과정에서의 우리의 컨벤션을 정할 수 있을 거라 생각해요!! 다시 한 번 미안합니다. 

이 코멘트에서 token을 주장한 것은 userToken이라고 작성하게 되면 PinService가 User라는 도메인을 알게 된다고 생각했어요! Pin 도메인에서 User라는 도메인을 알아야 할까? 오히려 유지보수 포인트만 늘어나는 거 아닐까 했어요. 만약에 우리가 User라는 네이밍에서 Member로 수정한다면 해당 메서드의 인자는 memberToken으로 변경해야 할거예요. 하지만 변경해야 함을 인지하기 어렵겠죠. pin은 외부 도메인이 User든 Member든 그 외의 어떤 것이든 알 필요 없이 token에만 의존하는 것이 좋다고 생각했어요. 

이해가 됐을까요?? 이해하기 어려운 코멘트라면 저에게 피드백을 주세요 🙇 그리고 이에 대한 수민이의 생각을 들려주세요~~

Reply:
아항 네이밍에서도 역할 분리가 있어야겠네요
이해했어요
앞으로 네이밍할 때 염두에 두겠습니다!~!🐝🐥🐢